### DISCUSSION TITLE
Proof of academic status denied

### DISCUSSION THREAD

**Discussion by Seanji2002** [community/community#43746] [GitHub Education]
Hi,

I have been trying to apply for the student development pack and my friends around me can get theirs but for some reason mine does not get approved. I do not have a dated student ID (on our student id there is no date) so I was trying an unofficial transcript. However, the site keeps rejecting me. Attached is a screenshot of me attempting to upload it.
If I am not clear, I am getting rejected before I can even submit the application request
**If you are having the same problem, please upvote so that more people can see**

Thanks in advance for your help!
Shuyang Ji

![image](https://user-images.githubusercontent.com/59269351/211359982-8cb7d24e-96e2-4411-a175-1a4a2ffc4a73.png)

**Comment by Y02WK** [comment]
Having the same issue. Submitted a ticket to support that got closed, hoping there is some resolution for us

**Reply by Seanji2002** [reply]
Good to know someone is with me XD. My friends didn't have this issue :(

**Reply by cmumatt** [reply]
@Toastbroti @Seanji2002 @Y02WK Yes, it is pretty reasonable to expect that we should be able to reach out to a human for help when these automated systems make errors, which they do regularly and not always in ways that are fair.

My application has been in limbo since late last year -- it's just stuck, and the auto-close ticket e-mail tells me to come here and ask you for help.  Just like it asked you to come here and ask me for help with GitHub's system.

Totally crazy, like what are we as students supposed to do to fix GitHub's broken system?  It just doesn't work because (a) it's buggy and (b) it's based on some set of ideals that usually does not exist in the real world, particularly in academia.  Why is this suddenly our problem to figure out and fix?
